### PR TITLE
Bug 2096315: Create a patch im CMO for NodeClockNotSynchronising

### PR DESCRIPTION
**Pull request by lnguyen1401**
* [ ] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

The runbook PR: https://github.com/openshift/runbooks/pull/36/files
The related Jira card: https://issues.redhat.com/browse/OSD-8737

### CODE REVIEW

**Comment by simonpasquier** [comment]
/hold

We need https://github.com/openshift/runbooks/pull/36 to be merged first.
We'd need a formal agreement from @openshift/teams/sre-alert-sme that increasing the severity to `critical` is ok. Once it's approved, the alerting rule should be patched in [jsonnet/utils/sanitize-rules.libsonnet](https://github.com/openshift/cluster-monitoring-operator/blob/843ca82c018431b95dcfdd6c32bf734d8f53319b/jsonnet/utils/sanitize-rules.libsonnet#L121)

**Comment by lnguyen1401** [comment]
> /hold
> 
> We need [openshift/runbooks#36](https://github.com/openshift/runbooks/pull/36) to be merged first. We'd need a formal agreement from @openshift/teams/sre-alert-sme that increasing the severity to `critical` is ok. Once it's approved, the alerting rule should be patched in [jsonnet/utils/sanitize-rules.libsonnet](https://github.com/openshift/cluster-monitoring-operator/blob/843ca82c018431b95dcfdd6c32bf734d8f53319b/jsonnet/utils/sanitize-rules.libsonnet#L121)

**Comment by lnguyen1401** [comment]
Hello @simonpasquier , do you have an SOP to create a formal agreement ?

**Comment by fahlmant** [comment]
@simonpasquier We have agreement. What's the process to "formal" agreement here?

**Comment by lnguyen1401** [comment]
hello @simonpasquier , the run book has been merged. i made a patch for NodeClockNotSynchronising in jsonnet/utils/sanitize-rules.libsonnet

**Review by boranx — COMMENTED** [review]
/lgtm

Looks like the change is straightforward, anything blocking us from merging this?

**Comment by ritmun** [comment]
I'm not sure what the assignment needs to be. I followed this message posted by CI bot
 
image

File: jsonnet/utils/sanitize-rules.libsonnet
Comment (jan--f) [inline jsonnet/utils/sanitize-rules.libsonnet:167]: This jsonnet snippet doesn't have any effect, the ClockNotSynced alert is in the `node-exporter` group. You need something like this (with probably better formatting):
```suggestion
    ],
  },
  {
    name: 'node-exporter',
    rules: [
      {
        alert: 'NodeClockNotSynchronising',
        labels: {
          severity: 'critical',
        },
      },
```

**Review by jan--f — CHANGES_REQUESTED** [review]

**Comment by simonpasquier** [comment]
/retitle Bug 2096315: Create a patch im CMO for NodeClockNotSynchronising

**Pull request merged**
